### PR TITLE
Added 60 FPS patch to NFS Hot Pursuit 2 NTSC

### PR DESCRIPTION
**Pull request by J-McFc**

### CODE REVIEW

File: patches/SLUS-20362_1D2818AF.pnach
Comment (fox140cv) [inline patches/SLUS-20362_1D2818AF.pnach:1]: Could it be that you are missing the code?

Comment (J-McFc) [reply patches/SLUS-20362_1D2818AF.pnach:1]: Well yknow that would be it haha thank you.

File: patches/SLUS-20362_1D2818AF.pnach [hunk 1]
```diff
@@ -7,3 +7,13 @@ comment=Widescreen Hack by nemesis2000
 patch=1,EE,0010e994,word,46011702
 patch=1,EE,0032f6ec,word,3f400000
 patch=1,EE,0032f850,word,3f2aaaaa
+
+
+[60 FPS]
+author=asasega
+comment=Patches the game to run at 60 FPS
+(EE Overclock should be set to 130%. Higher values are usable but may cause hanging).
```
Comment (refractionpcsx2) [inline patches/SLUS-20362_1D2818AF.pnach:15]: Can you keep the comment on the same line, it'll confuse the parser

Comment (J-McFc) [reply patches/SLUS-20362_1D2818AF.pnach:15]: Sure

Comment (refractionpcsx2) [reply patches/SLUS-20362_1D2818AF.pnach]: Thanks :)

**Pull request merged**